c_binding/include/libstoragemgmt/libstoragemgmt.h

```diff
@@ -37,111 +37,217 @@
 #include "libstoragemgmt_local_disk.h"
 #include "libstoragemgmt_battery.h"
 
-
-/*! \mainpage libStorageMgmt
- *
- * \section Introduction
- *
- * The libStorageMgmt package is a storage array independent Application
- * Programming Interface (API). It provides a stable and consistent API that
- * allows developers the ability to programmatically manage different storage
- * arrays and leverage the hardware accelerated features that they provide.
- *
- *  \section additional Additional documentation
- *
- * Full documentation can be found at:
- * http://libstorage.github.io/libstoragemgmt-doc/
- *
- */
-
 #ifdef  __cplusplus
 extern "C" {
 #endif
 
 /**
- * Get a connection to a storage provider.
- * @param[in] uri       Uniform Resource Identifier (see URI documentation)
- * @param[in] password  Password for the storage array (optional, can be NULL)
- * @param[out] conn     The connection to use for all the other library calls
- * @param[in] timeout   Time-out in milliseconds, (initial value).
- * @param[out] e        Error data if connection failed.
- * @param[in] flags     Reserved for future use, must be zero.
- * @return Error code as enumerated by \ref lsm_error_number.
- * @retval LSM_ERR_OK on success.
+ * lsm_connect_password - Get a connection to a storage provider.
+ *
+ * Version:
+ *      1.0
+ *
+ * Description:
+ *      Get a connection to a storage provider.
+ *
+ * @uri:
+ *      Uniform Resource Identifier (see URI documentation)
+ * @password:
+ *      Password for the storage array (optional, can be NULL)
+ * @conn:
+ *      The connection to use for all the other library calls
```

Comment:
. When done using the connection it must be freed with a call to lsm_connect_close.